packages/react/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { ComponentProps } from 'react';
+
+import { Meta, StoryObj } from '@storybook/react';
+
+import { Checkbox } from '.';
+
+type Args = ComponentProps<typeof Checkbox>;
+
+const meta: Meta<Args> = {
+  tags: ['autodocs'],
+  component: Checkbox,
+  parameters: {
+    references: ['Checkbox'],
+  },
+  argTypes: {
+    checkedIcon: {
+      control: {
```

Comment:
Давай прям скроем их
```suggestion
      table: {
```